.github/workflows/cross-version-tests.yml

```diff
@@ -78,7 +78,8 @@ jobs:
           else
             python_version=3.6
           fi
-          echo "::set-output name=python-version::$python_version"
+          latest_micro_version=$($CONDA/bin/conda search python | grep -F $python_version | tail -1 | tr -s ' ' | cut -d' ' -f2)
```

Comment:
`conda search python` outputs:

```
python                        3.6.10      h359304d_0  pkgs/main           
python                        3.6.10      hc70fcce_1  pkgs/main           
python                        3.6.10      hf48f09d_2  pkgs/main           
python                        3.6.10      hfe9666f_1  pkgs/main           
python                        3.6.12      h26836e1_2  pkgs/main           
python                        3.6.13      h88f2d9e_0  pkgs/main           
python                         3.7.0      hc167b69_0  pkgs/main           
python                         3.7.1      haf84260_3  pkgs/main   
```

`grep -F $python_version`: Captures specified minor python versions
`tail -1`: Fetches the last row
`tr -s ' '`: Shrinks spaces
`cut -d' ' -f2`: Splits the row with a space and get the second element.